interface/fax/fax_dispatch.php

```diff
@@ -614,203 +594,230 @@ function allCheckboxes(issel) {
 </head>
 
 <body class="body_top" onunload='imclosing()'>
-
-<center><h2><?php echo xlt('Dispatch Received Document'); ?></h2></center>
+<h2 class="text-center"><?php echo xlt('Dispatch Received Document'); ?></h2>
```

Comment:
Hello @bradymiller  and @tywrenn 

I found the `onunload='imclosing()` will occur error when the Scanned Dispatch dialog close,
should I remove it?

![image](https://user-images.githubusercontent.com/22230889/83755353-0a89c280-a6a0-11ea-8e28-1a2f155da4c5.png)